### DISCUSSION TITLE
Ya está la práctica 0???

### DISCUSSION THREAD

**Discussion by samusisto** [INGCOM-UNRN-P1/cursada-2023#5] [Preguntas y respuestas]
Hola. Quiero saber si ya está la práctica CERO (?) y cuándo es la fecha de entrega?
(aclaro que me retiré antes y seguro me faltó info. No me odien jaja)

**Comment by martinvilu** [comment]
El enlace a la [Práctica #0](https://classroom.github.com/assignment-invitations/df27ecfb9dd312cde5084babe55b0028) está en el campus bimodal

https://campusbimodal.unrn.edu.ar/course/view.php?id=40610&section=1#sectionid-568649-title

Fecha de entrega: 7 de agosto

Fecha límite: 9 de agosto
